docs/tutorials/analysis/1D/spectral_analysis_hli.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>This is already coded in the <code>AnalysisConfig</code> yaml.  This is not needed here. Maybe simply show that one can access the content this way, but don't reset everything </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/4044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>removed</p>